client/src/pages/guide/english/html/tutorials/images-in-html/index.md

```diff
@@ -38,6 +38,11 @@ To insert an image you define the source and an alternative text wich is display
 <img src="URL of the Image" alt="Descriptive Title" style="width: 100px; height: 150px;"/>
 ```
 
+### Make rounded image:
```

Comment:
Since this is an example of styling, it should be a heading one level lower than the one before.